netbox-plugin.yaml

```diff
@@ -1,6 +1,9 @@
 version: 0.1
 package_name: netbox-topology-views
 compatibility:
+  - release: 4.2.0
+    netbox_min: 4.2.0
+    netbox_max: 4.2.1
   - release: 4.1.0
     netbox_min: 4.1.0
     netbox_max: 4.1.4
```

Comment:
This NTV-Version should work with all 4.1.x, up to 4.1.11. The README.md in this PR does reflect that fact.

Reply:
Updated the manifest to set the max version to 4.1.11